src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,62 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int maxColorValue = 5;
+    private static final int maxRandomValue = 10;
+    private static final String defaultColor = String.valueOf(ColorSupplier.Color.WHITE);
+    private static final int defaultRadius = 10;
+    private final Random random = new Random();
+    private final ColorSupplier colorSupplier = new ColorSupplier();
+
+    public Figure getRandomFigure() {
+        int randomNumber = random.nextInt(maxColorValue);
+
+        switch (randomNumber) {
+            case 0:
+                return getRandomCircle();
```

Comment:
```suggestion
                return getRandomCircle(randomColor);
```

Reply:
Same for other figures

Reply:
Sorry, but i dont get it - i understand that i should add argument in every method where i create a random figure and add this in swich case, but i dont understand how it improve the code, can u explain it please? 
thank you*

Reply:
Its need to avoid using "colorSupplier.getRandomColor( )" expression  in methods?